test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -111,15 +117,18 @@ test
     await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().innerText).eql('Back to sign in');
     await t.expect(loopbackSuccessLogger.count(
       record => record.response.statusCode === 200 &&
+      record.request.method !== 'options' &&
```

Comment:
With the latest version, the RequestLogger also tracks the pre-flights requests here - this is different from the previous version we were using before.